website/content/docs/configuration/events/telemetry-data.mdx

```diff
@@ -0,0 +1,586 @@
+---
+layout: docs
+page_title: Telemetry data
+description: >-
+  Learn what data Boundary captures when telemetry is enabled and how you can view it.
+---
+
+# Telemetry data
```

Comment:
Is this going to be an internal doc?

Reply:
No, Anando wants to publish these steps for our external users so they can benefit from telemetry. So we would publish this to developer.com with our other external docs.

Reply:
Got it thanks for clarifying Dan